tools/run_tests/run_interop_tests.py

```diff
@@ -1353,10 +1380,10 @@ def aggregate_http2_results(stdout):
                     server_port = _DEFAULT_SERVER_PORT + offset
                     if not args.manual_run:
                         server_port = http2_server_job.mapped_port(server_port)
-                    if not args.insecure:
+                    if args.transport_security != 'none':
                         print((
                             'Creating grpc cient to http2 server test case with insecure connection, even though'
```

Comment:
typo: s/cient/client/

Reply:
s/none/None/

Reply:
Ah, just realized the 'none' is a string constant - I'd say let's rename, because it's very easy to confuse with python's `None`.